lib/rules/no-unexpected-multiline.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @fileoverview Rule to spot scenarios where a newline looks like it is ending a statement, but is not.
+ * @author Glen Mailer
+ * @copyright 2015 Glen Mailer
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+module.exports = function(context) {
+
+    var FUNCTION_MESSAGE = "This ( is a function call against the previous line, which is misleading.";
```

Comment:
We need to tighten up the messages here, as we aren't fully explaining the problem. Maybe, "Unexpected lines between ( and function name."?